# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/rustc/mips.patch
# Copyright (C) 2022 - 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- rustc-1.69.0-src/compiler/rustc_target/src/spec/targets/mips_unknown_linux_gnu.rs.vanilla	2023-05-02 19:52:37.556686706 +0200
+++ rustc-1.69.0-src/compiler/rustc_target/src/spec/targets/mips_unknown_linux_gnu.rs	2023-05-02 19:54:43.886693884 +0200
@@ -9,8 +9,7 @@
         arch: "mips".into(),
         options: TargetOptions {
             endian: Endian::Big,
-            cpu: "mips32r2".into(),
-            features: "+mips32r2,+fpxx,+nooddspreg".into(),
+            cpu: "mips3".into(),
             max_atomic_width: Some(32),
             mcount: "_mcount".into(),
 
